### ISSUE TITLE
Invalid If-Modified-Since and If-None-Match processing

### ISSUE THREAD

**Issue by ForNeVeR** [ForNeVeR/fornever.me#25]
_(Extracted from #22.)_

The following request seems to be processed incorrectly.

```
Host: fornever.me
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64; rv:42.0) Gecko/20100101 Firefox/42.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Connection: keep-alive
If-Modified-Since: Sun, 01 Nov 2015 12:27:27 GMT
If-None-Match: "5636052f-13b9"
Cache-Control: max-age=0
```

**Comment by ForNeVeR** [comment]
This request could demonstrate an issue:

    $ curl -H 'If-Modified-Since: Sun, 01 Nov 2015 12:27:27 GMT' -H 'If-None-Match: "5636052f-13b9"' -v https://fornever.me --insecure -o T:\Temp\file.txt

**Comment by ForNeVeR** [comment]
Closed with 423d8a33a2d7acfa61838c6c9c29afe51400f8aa.
